src/main/java/org/skriptlang/skript/bukkit/loottables/LootContextCreateEvent.java

```diff
@@ -0,0 +1,24 @@
+package org.skriptlang.skript.bukkit.loottables;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class LootContextCreateEvent extends Event {
+
+	private final LootContextWrapper context;
+
+	public LootContextCreateEvent(LootContextWrapper context) {
+		this.context = context;
+	}
+
+	public LootContextWrapper getWrapper() {
```

Comment:
I think I prefer this as `getLootContext` or `getContext` the usage of `getWrapper` just doesn't really explain what it returns